pkgs/applications/misc/rofi-emoji/default.nix

```diff
@@ -15,61 +15,58 @@
 , rofi-unwrapped
 , wl-clipboard
 , xclip
-, xsel
 , xdotool
 , wtype
 }:
 
-stdenv.mkDerivation rec {
-  pname = "rofi-emoji";
-  version = "3.4.0";
+import ./versions.nix ({ version, sha256, patches}:
+  stdenv.mkDerivation rec {
+    pname = "rofi-emoji";
+    inherit version;
 
-  src = fetchFromGitHub {
-    owner = "Mange";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-tF3yAKRUix+if+45rxg5vq83Pu33TQ6oUKWPIs/l4X0=";
-  };
+    src = fetchFromGitHub {
+      owner = "Mange";
+      repo = pname;
+      rev = "v${version}";
+      hash = sha256;
```

Comment:
nit: let's call this variable `hash`

Reply:
Then we can use `inherit hash`

Reply:
I'll change this!

I copied this pattern from some other `versions.nix` files. I assume I should not do the same change everywhere, but I'm raising it anyway in case it's something you want. :-)
(In a separate commit, of course)